src/cryptography/hazmat/backends/openssl/ocsp.py

```diff
@@ -306,6 +306,19 @@ def serial_number(self):
     def extensions(self):
         return _OCSP_BASICRESP_EXT_PARSER.parse(self._backend, self._basic)
 
+    def public_bytes(self, encoding):
+        if encoding is not serialization.Encoding.DER:
+            raise ValueError(
+                "The only allowed encoding value is Encoding.DER"
+            )
+
+        bio = self._backend._create_mem_bio_gc()
+        res = self._backend._lib.i2d_OCSP_RESPONSE_bio(
```

Comment:
Why are we using the `bio` variant instead of the normal `char *` one?

Reply:
🤷‍♀️ 

It's what we've got bound. We could switch the binding for both this and OCSP request though.